pilot/pkg/model/sidecar.go

```diff
@@ -80,6 +80,9 @@ type SidecarScope struct {
 	// XDSOutboundClusters is the CDS output for sidecars that map to this
 	// sidecarScope object. Contains the outbound clusters only
 	XDSOutboundClusters []*xdsapi.Cluster
+
+	// PerRouteFilterConfig has plugin Router Filter config
+	PerRouteFilterConfig map[string]interface{}
```

Comment:
RDSPerRouteFilterConfig

Reply:
it applies to TCP as well ? in that case, XDSPerRouteFilterConfig

Reply:
Done~